docs/en/Concept.md

```diff
@@ -0,0 +1,41 @@
+## 7 Pull Consumer
+ A type of Consumer.Applications are usually pulls messages from brokers by actively calling the Consumer's pull message method,and the application has the advantages of controlling the timing and frequency of pulling messages. Once batches of messages are pulled, user application initiates consuming process.                    
+## 8 Push Consumer
+ A type of Consumer.Under this mode, after the Broker receives the data, it will actively push it to the consumer, which is generally of high real-time performance.                       
+## 9 Producer Group
+  A collection of the same type of Producer, which sends the same type of messages with consistent logic.If a transaction message is sent and the original producer crashes after sending, the Broker Server will contacts other Producer in the same Producer group to commit or rollback  the transactional message.
+## 10 Consumer Group
+  A collection of the same type of Consumer, which sends the same type of messages with consistent logic.The Consumer Group makes load-balance and fault-tolerance super easy in terms of message consuming.
+Warning: consumer instances of one consumer group must have exactly the same topic subscription(s).   
+
+RocketMQ supports two type of consumption mode:Clustering and Broadcasting.
+## 11 Consumption Mode - Clustering
+Under the Clustering mode, each Consumer instance of the same Consumer Group allocates messages equally.
+## 12 Consumption Mode - Broadcasting
+Under the Broadcasting mode, each Consumer instance of the same Consumer Group receives a full number of messages.
+## 13 Normal Ordered Message
+Under the Normal Ordered Message mode, the messages received by consumers through the same consumption queue are sequential, while the messages received by different message queues may be non-sequential.
+## 14 Strictly Ordered Message
+Under the Strictly Ordered Message mode, all messages received by the consumer are sequential.
+## 15 Message
+The physical carrier of information transmitted by a messaging system, the smallest unit of production and consumption data, each message must belong to a topic.
+Each Message in RocketMQ has a unique Message ID and can carry a Key with a business identity.The system provides the function to query messages by Message ID and Key.
+## 16 Tag
+ Flags set for messages to distinguish different types of messages under the same topic.Messages from the same business unit can set different tag under the same topic in terms of different business purposes.The Tag can effectively maintain the clarity and consistency of the code and optimize the query system provided by RocketMQ.The Consumer can realize different consumption logic for different subtopics according to Tag, so as to achieve better expansibility.
```

Comment:
1. Flags set for messages to distinguish different types of messages under the same topic, **functioning as a "sub-topic"**.
2. Messages from the same business unit can set different **tags** under the same topic.
3. The Consumer can realize different consumption logic for different **"sub-topic"** **by using Tag**, so as to achieve better expansibility.